src/main/resources/oph-configuration/akt.properties.template

```diff
@@ -3,4 +3,5 @@ postgresql.username={{postgres_app_user}}
 postgresql.password={{host_postgresql_akt_app_password}}
 
 url-alb={{host_alb}}
-ryhmasahkoposti-service=${url-alb}/ryhmasahkoposti-service/email/firewall
\ No newline at end of file
+ryhmasahkoposti-service=${url-alb}/ryhmasahkoposti-service/email/firewall
+email.sending.enabled=true
```

Comment:
Nämä voi halutessa vaihtaa päittäin jotta oisivat määritelty samassa järjestyksessä kuin application.yaml:ssa. Toistaiseksi ei mitään väliä mut jos konffeja olis paljon niin ois kätsy.